src/test/java/us/abstracta/wiresham/VirtualTcpServiceTest.java

```diff
@@ -115,6 +116,26 @@ public void shouldGetExpectedResponseWhenConnectSsl() throws Exception {
     waitReceived("Hello");
   }
 
+  @Test
```

Comment:
Consider moving this test to a new FlowTest class and rename it to be more explicit about the functional intent of the functionality, maybe something like `shouldGetProperStepTypesWhenLoadWiresharkWithSameIpsForServerAndClientAndPortIsSpecified`. Another possibility might be just change this test to something like shouldGetExpectedResponseWhenInputWithServerAndClientWithSameIp and test that you get expected behavior when sending bytes to the mock.

If you prefer the first option, consider also using something like:

```
List<Class<?>> stepsTypes = flow.getSteps().stream().map(PacketStep::getClass).collect(Collectors.toList());
    assertEquals(stepsTypes, Arrays.asList(ClientPacketStep.class, ServerPacketStep.class,ClientPacketStep.class,ServerPacketStep.class));
```

Which is shorter, and allows you to detect more issues than current form which will only report the first detected issue.